python/ray/data/read_api.py

```diff
@@ -1755,6 +1755,86 @@ def create_connection():
     )
 
 
+@PublicAPI(stability="alpha")
+def read_databricks_uc_tables(
```

Comment:
I like the name for now; in the future, it'll be great to have `read_delta` without `warehouse_id` :D